internal/app/proxyd/internal/frontend/frontend.go

```diff
@@ -43,18 +36,13 @@ type ReverseProxy struct {
 }
 
 // NewReverseProxy initializes a ReverseProxy.
-func NewReverseProxy(endpoints []string) (r *ReverseProxy, err error) {
-	config, err := tlsConfig()
```

Comment:
I'm not sure about this just yet. I think we still need this to make sure that the initial request isn't send over the wire unencrypted.

Reply:
This is dropped because reverseproxy.Config doesn't actually get used anywhere. We generate a tlsConfig, but don't actually use it.